Account.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+
+public class Account
+{
+	private string name;
+	private decimal balance;
+
+	public List<Transaction> transactions { get; }
+
+	public Account(string name)
+	{
+		this.name = name;
+		transactions = new List<Transaction>();
+	}
+
+	public void AddTransaction(Transaction newTrans)
```

Comment:
It's best practice to avoid abbreviations when naming. It can make the code harder to read as people don't always know/use the same abbreviations 🙂